org.azahar_emu.Azahar.json

```diff
@@ -0,0 +1,143 @@
+                    "sha256": "9fe736980d424c04f1303ae71b94b18bcc6046ae348909c393344a45e1bd7ff8",
+                    "dest": "External/spirv-tools"
+                },
+                {
+                    "type": "archive",
+                    "url": "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.4.304.1.tar.gz",
+                    "sha256": "66e6cec19e7433fc58ace8cdf4040be0d52bb5920e54109967df2dd9598a8d48",
+                    "dest": "External/spirv-tools/external/spirv-headers"
+                }
+            ]
+        },
+        {
+            "name": "rapidjson",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DRAPIDJSON_ENABLE_INSTRUMENTATION_OPT=OFF",
+                "-DRAPIDJSON_BUILD_DOC=OFF",
+                "-DRAPIDJSON_BUILD_EXAMPLES=OFF",
+                "-DRAPIDJSON_BUILD_TESTS=OFF",
+                "-DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz",
+                    "sha256": "bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "stable-only": true,
+                        "project-id": 7422,
+                        "url-template": "https://github.com/Tencent/rapidjson/archive/v$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "azahar",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_C_COMPILER=clang",
+                "-DCMAKE_CXX_COMPILER=clang++",
+                "-DCMAKE_LINKER=lld",
+                "-DENABLE_QT_TRANSLATION=ON",
+                "-DUSE_DISCORD_PRESENCE=ON",
+                "-DUSE_SYSTEM_SDL2=ON"
+            ],
+            "cleanup": [
+                "/share/man",
+                "/share/pixmaps"
+            ],
+            "post-install": [
+                "install -Dm755 ../azahar-launcher.sh /app/bin/azahar-launcher",
+                "install -Dm644 ../org.azahar_emu.Azahar.metainfo.xml /app/share/appdata/org.azahar_emu.Azahar.metainfo.xml",
+                "desktop-file-install --dir=/app/share/applications ../dist/azahar.desktop",
+                "desktop-file-edit --set-key StartupWMClass --set-value azahar --set-key Exec --set-value azahar-launcher /app/share/applications/azahar.desktop",
+                "install -Dm644 ../dist/azahar.svg /app/share/icons/hicolor/scalable/apps/azahar.svg",
+                "install -Dm644 ../dist/icon.png /app/share/icons/hicolor/512x512/apps/azahar.png"
```

Comment:
why aren't the icons installed by the build system?

Reply:
Same as the previous comment, but the 512x512 icon wasn't installed by the build system. I will add that upstream.